pkg/cli/status.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cli
+
+import (
+	"strings"
+
+	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+)
+
+type Info struct {
+	Key, Value interface{}
+}
+
+func StatusConditionInfo(cond *duckv1alpha1.Condition) []Info {
+	info := make([]Info, 5)
+	info[0] = Info{"Type", cond.Type}
+	info[1] = Info{"Status", cond.Status}
+	info[2] = Info{"Reason", cond.Reason}
+	info[3] = Info{"Severity", cond.Severity}
+	info[4] = Info{"Message", cond.Message}
+	return info
+}
+
+func PrintStatusMessage(c *Config, resource string, info []Info, blankLine bool) {
+	if blankLine {
+		c.Printf("\n")
+	}
+	c.Printf("%s:\n", strings.ToUpper(resource))
+	for i := 0; i < len(info); i++ {
+		c.Printf(" %s: %s\n", info[i].Key, info[i].Value)
```

Comment:
consider using a [tabwriter](https://github.com/projectriff/riff/blob/master/pkg/cli/printers/tabwriter/tabwriter.go) to create vertical columns between the keys and values.